config/locales/subscriber_authentication.yml

```diff
@@ -18,6 +18,14 @@ en:
         message: "Enter your email address"
     request_sign_in_token:
       heading: "Check your inbox"
-      confirmation: "We’ve sent an email to %{address}."
-      prompt: Click the link in the email to manage your subscriptions.
-      delay_warning: It can take up to 15 minutes for the email to arrive.
+      description_html: |
+        <p>We’ve sent an email to <strong>%{address}</strong></p>
+        <br>
+        <p>Click the link in the email to confirm your email address.</p>
+        <br>
+        <p>The link will stop working after 7 days.</p>
```

Comment:
This doesn't feel a semantic use of br - looks like it's being used to achieve styling. Is there a margin bottom modifier class that can be used instead?

Reply:
Oh interesting. I'd always thought of `<br>` as a stylistic element, and the Internet jury is vague on this point. But I can see that semantically it makes more sense for it to only be used within a `<p>`.

I can see it's really GOV.UK confusing things here, by removing the blank lines that normal follow `<p>` elements. I've changed the 5th commit to apply the `govuk-body` class, which restores the spacing.